server/lib/nl/detection/palm_api.py

```diff
@@ -0,0 +1,336 @@
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Interface to PaLM API for detection"""
+
+import json
+import logging
+import time
+from typing import Dict
+
+from flask import current_app
+import requests
+
+from server.lib.nl.common import counters
+
+_API_URL_BASE = f"https://generativelanguage.googleapis.com/v1beta2/models/chat-bison-001:generateMessage"
+_API_HEADER = {'content-type': 'application/json'}
+
+
+_PROMPT_CONTEXT_ORIG = \
+("""You are going to be a super helpful assistant that will convert an english sentence to a JSON object.
```

Comment:
do you want to have a text file to store the prompt only?

Reply:
Good idea.  I'll do in followup PR (added TODO).

I'd then need to add load it into config in __init__.py and use it here right?